src/index.html

```diff
@@ -5,6 +5,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@Dubby20 why are we adding Fontawesome?

Reply:
I used it in styling the notification icon
<img width="101" alt="Screenshot 2019-05-03 at 2 30 10 PM" src="https://user-images.githubusercontent.com/29398098/57140667-3907e080-6db0-11e9-9707-510bbafb581f.png">